--- robot_models/yamax.orig.urdf	2018-10-31 23:16:36.000000000 +0900
+++ robot_models/yamax.urdf	2018-11-05 05:02:37.699771059 +0900
@@ -13,7 +13,6 @@
   <material name="Black">
     <color rgba="0.1718 0.2421 0.3125 1.0"/>
   </material>
-  <link name="base_link"/>
   <link name="hip">
     <visual>
       <origin rpy="0 0 0" xyz="0 0 0"/>
@@ -901,10 +900,6 @@
     <self_collide>False</self_collide>
     <material>Gazebo/Orange</material>
   </gazebo>
-  <joint name="dummy_joint" type="fixed">
-    <parent link="base_link"/>
-    <child link="hip"/>
-  </joint>
   <gazebo>
     <plugin filename="libgazebo_ros_p3d.so" name="ground_truth">
       <robotNamespace>/yamax</robotNamespace>
